src/app/components/announcements/announcements-fab-wrapper.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import { AxiosProvider } from 'api/axios';
+import AppWrapper from 'app-wrapper';
+import ChplAnnouncementsFab from './announcements-fab';
+
+const queryClient = new QueryClient();
+
+function ChplAnnouncementsFabWrapper() {
+  return (
+    <AppWrapper>
+      <AxiosProvider>
+        <QueryClientProvider client={queryClient}>
+          <ChplAnnouncementsFab />
+        </QueryClientProvider>
+      </AxiosProvider>
+    </AppWrapper>
```

Comment:
```suggestion
    <AppWrapper>
          <ChplAnnouncementsFab />
    </AppWrapper>
```

The AppWrapper includes an AxiosProvider and a QueryClientProvider. This code sets up to have two of each of those